tests/mail/tests.py

```diff
@@ -1122,3 +1124,55 @@ def test_server_stopped(self):
             backend.close()
         except Exception as e:
             self.fail("close() unexpectedly raised an exception: %s" % e)
+
+
+class SendTestEmailManagementCommand(SimpleTestCase):
```

Comment:
@timgraham to create it in this directory I should create a subdir called `tests` and have two files there `test_sendmail.py` and `test_mailbackends.py` or something similar, right?